api/v1beta1/flinkcluster_types.go

```diff
@@ -434,6 +435,18 @@ type JobManagerIngressStatus struct {
 	URLs []string `json:"urls,omitempty"`
 }
 
+// JobManagerServiceStatus defines the observed state of FlinkCluster
+type JobManagerServiceStatus struct {
+	// The name of the Kubernetes jobManager service.
+	Name string `json:"name"`
+
+	// The state of the component.
+	State string `json:"state"`
+
+	// The port of NodePort.
+	Port int32 `json:"port,omitempty"`
```

Comment:
I feel it is better to rename it to `NodePort` and add comment to mention it is only set when the AccessScope is NodePort, otherwise people might confuse it with the static port 8081.